torchtext/experimental/datasets/raw/sequence_tagging.py

```diff
@@ -0,0 +1,134 @@
+import torch
+
+from torchtext.utils import download_from_url, extract_archive
+
+URLS = {
+    "UDPOS":
+    'https://bitbucket.org/sivareddyg/public/downloads/en-ud-v2.zip',
+    "CoNLL2000Chunking": [
+        'https://www.clips.uantwerpen.be/conll2000/chunking/train.txt.gz',
+        'https://www.clips.uantwerpen.be/conll2000/chunking/test.txt.gz'
+    ]
+}
+
+
+def _create_data_from_iob(data_path, separator="\t"):
```

Comment:
Do we return or yield something from this func?

Reply:
yes, please take a look at line 22. due to the nature of the data, each sentences is separated with an empty line, therefore we will return one sentence if we found one. and I just add new commit to return leftovers